_data/changelogs/docs-getting-started-devs-phase-2.yml

```diff
@@ -0,0 +1,112 @@
+title: "Phase 2: Compile"
+type: documentation
+changelogURL:
+items:
+  - date: 2022-10-11
+    summary: Restructured content to add clarity.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1766
+    githubRepo: uswds-site
+  - date: 2022-08-16
+    summary: Fixed typo by replacing `_index.html` with `_index.scss`.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1740
+    githubRepo: uswds-site
+  - date: 2022-06-17
+    summary: Added warning and success note for `gulp init`.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1667
+    githubRepo: uswds-site
+  - date: 2022-06-03
+    summary: Updated the example compile settings with values that do not generate warnings.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1597
+    githubRepo: uswds-site
+  - date: 2022-04-28
+    summary: Updated documentation to use Sass module syntax and updated settings table.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1538
+    githubRepo: uswds-site
+  - date: 2022-03-10
+    summary: Replaced `paths.dist.sass` with `paths.dist.theme`.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1473
+    githubRepo: uswds-site
+  - date: 2022-03-10
+    summary: Replaced `uswds-gulp` with `uswds-compile`.
```

Comment:
I'm wondering if people will understand that this is a replacement to the text on the page rather than the actual file?  See line 18 for an example of how I would expect this to read if it were just about text:

"Updated text to replace 'uswds-gulp' with 'uswds-compile'.

Reply:
Updated according to the suggestion in https://github.com/uswds/uswds-site/pull/1919#discussion_r1037584948

Updated in: https://github.com/uswds/uswds-site/pull/1919/commits/afe4b145041a1e2f999bf84ce6cdcf5c6c39f208